### DISCUSSION TITLE
How to set ZooKeeper service as nodeport and set the port number

### DISCUSSION THREAD

**Discussion by TalZiv** [strimzi/strimzi-kafka-operator#9076] [Q&A]
Hi,

I am setting up a Kind cluster with Kafaka and I want to map the Kafka and ZooKeeeper port to the host.
I have managed to set nodeport and port numbers for the brokers and expose them to the local machine but not for ZooKeeper.
I did not find any reference in the documentation for that option.

Any help would be highly appreciate.
Thanks.

**Comment by scholzj** [comment]
Strimzi does not support exposing ZooKeeper and it should not be needed for anything. You should be able to do everything through Kafka APIs.

Long time ago I wrote this tool for accessing ZooKeeper: https://github.com/scholzj/zoo-entrance => if you want, feel free to use it. You should be able to just change the service type. But keep in mind it has no security and it's not an official part of Strimzi.
